x-pack/test/functional/services/ml/data_frame_analytics_creation.ts

```diff
@@ -110,6 +118,28 @@ export function MachineLearningDataFrameAnalyticsCreationProvider(
       );
     },
 
+    async assertJobDescriptionEditValue(expectedValue: string) {
+      const actualJobDescription = await testSubjects.getAttribute(
+        'mlAnalyticsEditFlyoutDescriptionInput',
+        'value'
+      );
+      expect(actualJobDescription).to.eql(
+        expectedValue,
+        `Job description edit should be '${expectedValue}' (got '${actualJobDescription}')`
+      );
+    },
+
+    async assertJobMmlEditValue(expectedValue: string) {
+      const actualMml = await testSubjects.getAttribute(
+        'mlAnalyticsEditFlyoutmodelMemoryLimitInput',
```

Comment:
nit: not sure if `model` is meant to be capitalized but if it's intentional then please disregard this comment